internal/log/log.go

```diff
@@ -20,3 +24,54 @@ func Logger() *logrus.Logger {
 
 // L is a convenience alias to Logger.
 var L = Logger
```

Comment:
do we still need this ?

Reply:
@komish is correct. I just wanted to get the conversation started without breaking the whole thing.